client/src/unifycr.c

```diff
@@ -147,15 +147,15 @@ int unifycr_initialized = 0;
 int unifycr_debug_level;
 
 /* global persistent memory block (metadata + data) */
-void *unifycr_superblock = NULL;
-static void *free_fid_stack = NULL;
-void *free_chunk_stack = NULL;
-void *free_spillchunk_stack = NULL;
-unifycr_filename_t *unifycr_filelist    = NULL;
-static unifycr_filemeta_t *unifycr_filemetas   = NULL;
-static unifycr_chunkmeta_t *unifycr_chunkmetas = NULL;
-
-char *unifycr_chunks = NULL;
+void* unifycr_superblock;
```

Comment:
Looks like `NULL` was removed from variables that are declared and initialized at the same time (here and in other places). Not sure if that is a problem or was intended.

Reply:
checkpatch requires that all "=NULL" initializations for global pointers to be dropped.

Reply:
Well quotes need some help, too...
```
+ /g/g0/moody20/projects/unifycr2.git/install/bin/unifycrd
# accepting RPCs on address "na+sm://146417/0"
completed sock init server
MLOG CRIT Error opening leveldb database, abc..., path is "/tmp"/unifycr_db-0-0
MLOG CRIT MDHIM Rank: 0 - Error while opening database
MLOG CRIT Rank: 0 - Error opening data store for index: 0
MLOG CRIT Error opening leveldb database, abc..., path is "/tmp"/unifycr_db-1-0
MLOG CRIT MDHIM Rank: 0 - Error while opening database
MLOG CRIT Rank: 0 - Error opening data store for index: 1
```

Reply:
@adammoody do we have to use macros for these paths?